avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastSerdeCache.java

```diff
@@ -305,14 +305,11 @@ private String getSchemaKey(Schema writerSchema, Schema readerSchema) {
    * @return a fast deserializer
    */
   public FastDeserializer<?> buildFastSpecificDeserializer(Schema writerSchema, Schema readerSchema) {
-    LOGGER.info("Start buildFastSpecificDeserializer for reader schema: [" + readerSchema + "] and write schema: ["
-        + writerSchema + "]");
-
     FastSpecificDeserializerGenerator<?> generator =
         new FastSpecificDeserializerGenerator<>(writerSchema, readerSchema, classesDir, classLoader,
             compileClassPath.orElseGet(() -> null));
     LOGGER.info("Generated classes dir: " + classesDir + ", and generation of specific FastDeserializer"
```

Comment:
is this enough string concats to use {} ?

Reply:
Those are not in the critical paths, so I guess either way is fine, and  using {}  should  be more  efficient, and let me try to adopt it  here.